src/SpellsReference/Models/Spell.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace SpellsReference.Models
+{
+    public class Spell
+    {
+        public Spell() { }
+
+        public Spell(string name, int level, SchoolOfMagic school, string castingTime, string range, 
+            bool verbal, bool somatic, string materials, string duration, string description)
+        {
+            Name = name;
+            Level = level;
+            School = school;
+            CastingTime = castingTime;
+            Range = range;
+            Verbal = verbal;
+            Somatic = somatic;
+            Materials = materials;
+            Duration = duration;
+            Description = description;
+        }
```

Comment:
I usually use it in the create function, but I copied your style of creating a new object in the code, instead of using the construction. Will remove this for now.